kube/services/fence/fence-deploy.yaml

```diff
@@ -222,6 +228,10 @@ spec:
             readOnly: true
             mountPath: "/fence/jwt-keys.tar"
             subPath: "jwt-keys.tar"
+          - name: "amanuensis-jwt-keys"
+            readOnly: true
+            mountPath: "/amanuensis/jwt_public_key.pem"
```

Comment:
I don't think the /amanuensis dir was created in the Dockerfile so this will probably fail. Please create the folder in the Dockerfile if you want to place this there.